ceph.spec.in

```diff
@@ -323,6 +324,8 @@ Requires:	python-flask
 %if 0%{?suse_version}
 # python-Flask for powerdns
 Requires:      python-Flask
+# libopenssl for civetweb https support
+Requires:      libopenssl1_0_0
```

Comment:
Is the explicit Requires needed here? The BuildRequires should be enough to see it added automatically, or am I missing something?